src/modules/graphics/Texture.h

```diff
@@ -194,6 +195,7 @@ class Texture : public Drawable, public Resource
 		bool renderTarget = false;
 		bool computeWrite = false;
 		OptionalBool readable;
+		Optional<std::string> debugName;
```

Comment:
This could be a `std::string` without the Optional wrapper, and then the backend code could check whether the string is empty before calling the graphics API to set it.